docs/src/user-guide/install.md

```diff
@@ -1,40 +1,111 @@
-# 💾 Install Anoma
+# 💾 Install Namada
```

Comment:
Let's not rename Anoma to Namada in the other section of the docs here (on the testnet page however it's fine)